charmcraft.yaml

```diff
@@ -140,6 +140,10 @@ config:
         This value should be equal to or shorter than half the root-ca-validity.
         Changing this value will trigger generation of a new CA certificate,
         revoking all previously issued certificates.
+    certificate-limit:
+      type: int
+      description: >
+        Maximum number of certificates that can be issued to a single requirer.
```

Comment:
```suggestion
        Maximum number of certificates that can be issued to a single requirer. Use `-1` for allowing an infinite amount of certificates.
```

Reply:
I think it's better for users to explicitly select the value when they want to move this value to infinity. 

This change (use of `-1`) will require some code changes of course.

Reply:
✅